fili-wikipedia-example/entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh 
+
+export HOSTIP="$(resolveip -s $HOSTNAME)" 
+/usr/bin/supervisord -c /etc/supervisor/conf.d/supervisord.conf & 
+echo "Waiting for Druid to finish setting up"
```

Comment:
I'm not going to block on this, but we *may* want to add a timeout here, so that docker doesn't just hang if Druid is having a problem loading up.